src/applications/vaos/services/appointment/transformers.js

```diff
@@ -181,9 +180,9 @@ function setVideoData(appt) {
     kind: videoData.appointmentKind,
     url: getVideoVisitLink(appt),
     isAtlas: !!videoData.tasInfo,
-    atlasLocation: videoData.tasInfo
-      ? transformATLASLocation(videoData.tasInfo)
-      : null,
+    // atlasLocation: videoData.tasInfo
```

Comment:
This is just to allow unit tests to pass. All v0 appointment transformer functions were removed in another PR that has not been merged yet: https://github.com/department-of-veterans-affairs/vets-website/pull/25018

Reply:
Removed this after merging in changes from `main`